ksql-common/src/main/java/io/confluent/ksql/util/KsqlConstants.java

```diff
@@ -56,4 +56,6 @@ private KsqlConstants() {
   public static final String AVRO_SCHEMA_NAME = "KsqlDataSourceSchema";
   public static final String DEFAULT_AVRO_SCHEMA_FULL_NAME =
           AVRO_SCHEMA_NAMESPACE + "." + AVRO_SCHEMA_NAME;
+
+  public static final String CONFLUENT_AUTHOR = "Confluent";
```

Comment:
i realize you only copied this author specification from the original PR for these functions but i've stopped adding it for buit-lin ones as it seems to add little/no value there. We should probably try to be consistent about that and leave this for 3rd party authors.

Reply:
Why not add `Confluent` or `built-in` to differentiate our UDFs from those written by others who have not set an author?